### DISCUSSION TITLE
Warning Message while Running

### DISCUSSION THREAD

**Discussion by spiralj** [PromtEngineer/localGPT#122] [General]
There's a warning now when running: UserWarning: You have modified the pretrained model configuration to control generation. This is a deprecated strategy to control generation and will be removed soon, in a future version. Please use a generation configuration file (see https://huggingface.co/docs/transformers/main_classes/text_generation)

**Comment by PromtEngineer** [comment]
@spiralj can you please confirm if this warning is resolved with the new update?

**Comment by spiralj** [comment]
Yep, all good.
